packages/wonder-blocks-form/src/__tests__/custom-snapshot.test.tsx

```diff
@@ -5,60 +5,45 @@ import CheckboxCore from "../components/checkbox-core";
 import RadioCore from "../components/radio-core";
 
 const states = ["default", "error", "disabled"];
-const clickableStates = ["default", "hovered", "pressed"];
 const checkedStates = [false, true];
 
 describe("CheckboxCore", () => {
     states.forEach((state: any) => {
-        clickableStates.forEach((clickableState: any) => {
-            checkedStates.forEach((checked: any) => {
-                test(`type:${state} state:${clickableState} checked:${String(
-                    checked,
-                )}`, () => {
-                    const disabled = state === "disabled";
-                    const tree = renderer
-                        .create(
-                            <CheckboxCore
-                                checked={checked}
-                                disabled={disabled}
-                                error={state === "error"}
-                                hovered={clickableState === "hovered"}
-                                pressed={clickableState === "pressed"}
-                                focused={clickableState === "focused"}
-                                waiting={false}
-                            />,
-                        )
-                        .toJSON();
-                    expect(tree).toMatchSnapshot();
-                });
+        checkedStates.forEach((checked: any) => {
+            test(`type:${state} checked:${String(checked)}`, () => {
+                const disabled = state === "disabled";
+                const tree = renderer
```

Comment:
**off-topic:** We should probably update this to use React Testing Library at some point.  Not req'd for this PR.